### PR TITLE
Remove periods in technical overview list

### PR DESCRIPTION
**Pull request by redbassett**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Documentation Update

## Description
This PR removes periods that were present at the end of only some lines in the technical overview list.

## Related Tickets & Documents
N/A

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
N/A

## Added to documentation?

- [x] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @redbassett, thanks for the contribution. Unfortunately there's a conflict because another PR with modifications on that file has been merged previously. You can use the "Resolve conflicts" button above to update your PR.

**Comment by redbassett** [comment]
That'd be me, I had another PR for this same file that got merged. Fixed!

File: docs/technical-overview.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ For the Dev.to tech stack we use:
...
+- [_Figaro_](https://github.com/laserlemon/figaro) for app configuration
 - [_CounterCulture_](https://github.com/magnusvk/counter_culture) to keep track of association counts (counter caches)
-- [_Rolify_](https://github.com/RolifyCommunity/rolify) for role management.
-- [_Pundit_](https://github.com/varvet/pundit) for authorization.
+- [_Rolify_](https://github.com/RolifyCommunity/rolify) for role management
+- [_Pundit_](https://github.com/varvet/pundit) for authorization
 - [_Service Workers_](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) to proxy traffic
-- [Preact](https://preactjs.com/) for some of the frontend. See [the Frontend Guide](/frontend) for more info
-- [_Pusher_](https://pusher.com) for realtime communication between the application and users' browsers.
+- [Preact](https://preactjs.com/) for some of the frontend – See [the Frontend Guide](/frontend) for more info
```
Comment (rhymes) [inline docs/technical-overview.md]: I believe this period was correct though :D

**Comment by redbassett** [comment]
Ignore the mistaken push that added unwanted commits, PR updated to revert that change.

File: docs/technical-overview.md (hunk 1 above)
Comment (redbassett) [reply docs/technical-overview.md]: Grammatically correct, but it bugs me. Reverted ;)

**Comment by redbassett** [comment]
Welp, that got messy. Actually updated this time.

**Pull request merged**